app/config/templates/site.php

```diff
@@ -1144,7 +1145,201 @@ function getFramework(string $frameworkEnum, array $overrides)
+            ]),
+        ],
+        'vcsProvider' => 'github',
+        'providerRepositoryId' => 'templates-for-sites',
+        'providerOwner' => 'appwrite',
+        'providerVersion' => '0.3.*',
+        'variables' => [],
+    ],
+    [
+        'key' => 'playground-for-vite',
+        'name' => 'Vite playground',
+        'tagline' => 'A basic Vite website without Appwrite SDK integration.',
+        'score' => 1, // 0 to 10 based on looks of screenshot (avoid 1,2,3,8,9,10 if possible)
+        'useCases' => [UseCases::STARTER],
+        'screenshotDark' => $url . '/images/sites/templates/playground-for-vite-dark.png',
+        'screenshotLight' => $url . '/images/sites/templates/playground-for-vite-light.png',
+        'frameworks' => [
+            getFramework('VITE', [
+                'providerRootDirectory' => './vite/starter',
+            ]),
+        ],
+        'vcsProvider' => 'github',
+        'providerRepositoryId' => 'templates-for-sites',
+        'providerOwner' => 'appwrite',
+        'providerVersion' => '0.3.*',
+        'variables' => [],
+    ],
+    [
+        'key' => 'playground-for-angular',
+        'name' => 'Angular playground',
+        'tagline' => 'A basic Angular website without Appwrite SDK integration.',
+        'score' => 1, // 0 to 10 based on looks of screenshot (avoid 1,2,3,8,9,10 if possible)
+        'useCases' => [UseCases::STARTER],
+        'screenshotDark' => $url . '/images/sites/templates/playground-for-angular-dark.png',
+        'screenshotLight' => $url . '/images/sites/templates/playground-for-angular-light.png',
+        'frameworks' => [
+            getFramework('ANGULAR', [
+                'providerRootDirectory' => './angular/starter',
+                'outputDirectory' => './dist/starter/browser',
+            ]),
+        ],
+        'vcsProvider' => 'github',
+        'providerRepositoryId' => 'templates-for-sites',
+        'providerOwner' => 'appwrite',
+        'providerVersion' => '0.3.*',
+        'variables' => [],
+    ],
+    [
+        'key' => 'playground-for-analog',
+        'name' => 'Analog playground',
+        'tagline' => 'A basic Analog website without Appwrite SDK integration.',
+        'score' => 1, // 0 to 10 based on looks of screenshot (avoid 1,2,3,8,9,10 if possible)
+        'useCases' => [UseCases::STARTER],
+        'screenshotDark' => $url . '/images/sites/templates/playground-for-analog-dark.png',
+        'screenshotLight' => $url . '/images/sites/templates/playground-for-analog-light.png',
+        'frameworks' => [
+            getFramework('ANALOG', [
+                'providerRootDirectory' => './analog/starter',
+            ]),
+        ],
+        'vcsProvider' => 'github',
+        'providerRepositoryId' => 'templates-for-sites',
+        'providerOwner' => 'appwrite',
+        'providerVersion' => '0.3.*',
+        'variables' => [],
+    ],
+    [
+        'key' => 'playground-for-svelte',
```

Comment:
should we call it `playground-for-sveltekit`?

Reply:
Sticking with svelte for now, same as starter